course/src/main/java/org/openedx/course/presentation/ui/CourseUI.kt

```diff
@@ -915,6 +916,57 @@ fun SubSectionUnitsList(
     }
 }
 
+@Composable
+fun CourseDatesBanner(
+    modifier: Modifier,
+    banner: CourseDatesBannerInfo,
+    resetDates: () -> Unit,
+) {
+    val cardModifier = modifier
+        .background(MaterialTheme.appColors.cardViewBackground, MaterialTheme.shapes.medium)
+        .border(1.dp, MaterialTheme.appColors.cardViewBorder, MaterialTheme.shapes.medium)
```

Comment:
As per figma the card has 8.dp radius corners

Reply:
we can use `MaterialTheme.appShapes.material.medium` for this

Reply:
It was a hindsight on my end. Thanks for pointing it out. 👍🏻